Tests/Marketplace/search_and_install_packs.py

```diff
@@ -434,45 +368,54 @@ def search_pack_and_its_dependencies(client: demisto_client,
             Each list contain one pack and its dependencies.
 
     """
-    pack_data = {}
-    if pack_id not in packs_to_install:
-        pack_display_name = get_pack_display_name(pack_id)
-        if pack_display_name:
-            pack_data = search_pack(client, pack_display_name, pack_id, lock)
```

Comment:
Why you deleted this function?

Reply:
The display name returned by`get_pack_display_name` was used in the past in the API call, but the code has changed since then, and now it's only used in logs returned by `search_pack` (which was removed).

`search_pack` is an API call that is used only to fetch the latest version of the pack, which is then passed to the dependencies API call.

This is redundant, since if we just pass a pack's ID without a version to the dependencies API call (on `get_pack_dependencies`), we already get the latest version.

Both of these functions are not in use anywhere else in the code.